modules/navier_stokes/include/tests/WedgeFunction.h

```diff
@@ -0,0 +1,48 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef WEDGEFUNCTION_H
+#define WEDGEFUNCTION_H
+
+#include "Function.h"
+#include "FunctionInterface.h"
+
+class WedgeFunction;
+
+template<>
+InputParameters validParams<WedgeFunction>();
+
+/**
+ * Function object for tests/ins/jeffery_hamel responsible for setting
+ * the inlet velocity components.  Inherits from FunctionInterface
+ * (similarly to CompositeFunction) so that it can couple to a
+ * PiecewiseLinear function which corresponds to the non-dimensional
+ * semi-analytic solution on the domain [0,1].  This function is
+ * responsible for scaling and mapping the semi-analytic solution
+ * values to the actual velocities used in the problem.
+ */
+class WedgeFunction : public Function,
+                      protected FunctionInterface
+{
+public:
+  WedgeFunction(const InputParameters & parameters);
+
+  virtual Real value(Real t, const Point & p);
+
+protected:
+  const Real _alpha_degrees;
+  const Real _Re;
+  const unsigned int _component;
+  const Real _mu;
+  const Real _rho;
+
```

Comment:
These should all be commented with /// or put a ///@{ block around and give them the same comment.